policy/resolver.go

```diff
@@ -1582,24 +1555,20 @@ func (s *LocalServices) jobsToControls(cache *frameworkResolverCache, framework
 				continue
 			}
 			uuid := cache.relativeChecksum(mquery.Mrn)
-			queryJob := &ReportingJob{
-				Uuid:      uuid,
-				QrId:      mquery.Mrn,
-				ChildJobs: map[string]*explorer.Impact{},
-				Type:      ReportingJob_DATA_QUERY,
-			}
-			nuJobs[uuid] = queryJob
-			for controlMrn := range targets {
-				controlJob := ensureControlJob(cache, nuJobs, controlMrn, framework, frameworkGroupByControlMrn)
```

Comment:
We no longer need this, there is code below that ties framework controls to its links (both queries and checks)